rethinkdb/content.md

```diff
@@ -0,0 +1,50 @@
+# What is RethinkDB?
+
+RethinkDB is an open-source, distributed database built to store JSON documents
+and effortlessly scale to multiple machines. It's easy to set up and learn and
+features a simple but powerful query language that supports table joins,
+groupings, aggregations, and functions.
+
+%%LOGO%%
+
+# How to use this image
+
+## start an instance with data mounted in the working directory
+
+The default CMD of the image is `rethinkdb --bind all`, so the RethinkDB daemon
+will bind to all network interfaces available to the container (by default,
+RethinkDB only accepts connections from `localhost`).
+
+```bash
+docker run --name some-rethink -v "$PWD:/data" -d rethinkdb
```

Comment:
This should be `-v "$(pwd):/data"`, or it could be `-v "/path/to/my/data:/data"` or even left out and the data will be preserved in a docker volume since "Volumes persist until no containers use them" ([docker docs](https://docs.docker.com/userguide/dockervolumes/)).